src/constant/database.ts

```diff
@@ -6,6 +6,8 @@ const databaseModels = {
   COURSE_SECTION: 'CourseSection',
   COURSE_CHAPTER: 'CourseChapter',
   USER_COURSE: 'UserCourse',
+  QUESTION: 'Question',
+  SHEET: 'Sheet',
```

Comment:
Name it properly. QUESTION should be INTERVIEW_QUESTION and SHEET should be INTERVIEW_SHEET.

But it's not required because simply Implement USER_INTERVIEW_SHEET that's all. Everything else is implemented.